README.md

```diff
@@ -58,3 +68,18 @@ public function calculate(DataProvider $dataProvider, float $multiplier)
 We have passed two arguments. One is `multiplier`. It is explicitly named. Such arguments passed as is. Another is 
 data provider. It is not named explicitly so injector finds matching parameter that has the same type.
 
+Creating an instance of an object of a given class behaves similar to `invoke()`:
+
+```php
+use Yiisoft\Injector\Injector;
+
+class StringFormatter
+{
+    public function __construct($string, \Yiisoft\I18n\MessageFormatterInterface $formatter)
```

Comment:
How about to use `DateTimeImmutable` instead of `MessageFormatterInterface`?

Reply:
DateTimeImmutable won't be stored in DI container. Thus, not a good example/